docs/plugins/contributors/gatsby-node.js

```diff
@@ -0,0 +1,70 @@
+const { Octokit } = require("@octokit/rest");
+const path = require("path");
+
+const NODE = `contributor`;
+
+exports.sourceNodes = async ({ actions, createContentDigest, createNodeId }, { repo, owner }) => {
+  const { createNode, createTypes } = actions;
+
+  createTypes(
+    `
+    type Contributor implements Node {
+      id: ID!
+      login: String
```

Comment:
Why include fields we're not querying for?

Reply:
it does not make any perfomance difference, if token is present, there are all fields there, i've added all possible we might use

Reply:
My concern would be that they could change the fields which would cause conflicts and breakage for things we don't need. I don't suppose it's all that likely, it just seemed unnecessary. 🤷